apps/api/views.py

```diff
@@ -112,3 +112,25 @@ def get_atis(request, icao):
         return JsonResponse(model_to_dict(atis))
     else:
         return HttpResponse(f'ATIS for facility {icao.upper()} was not found.', status=404)
+
+
+class TMUNotice(object):
+    pass
```

Comment:
Not how you make a model !!! Look at the other models or the [Django documentation](https://docs.djangoproject.com/en/3.1/topics/db/models/).